samza-core/src/main/java/org/apache/samza/config/ClusterManagerConfig.java

```diff
@@ -192,6 +198,11 @@ else if (containsKey(CONTAINER_RETRY_COUNT)) {
     }
   }
 
+  public boolean getContainerFailJobAfterRetries() {
```

Comment:
Should we add a javadoc here, 
and call it something like 
shouldFailJobAfterContainerRetries() ?

Reply:
Will add.